flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -132,6 +133,13 @@
 	 */
 	protected final OutputTag<IN> lateDataOutputTag;
 
+	/**
+	* Metrics about the lost data due to arrive late.
```

Comment:
after addressing the below comment about the renaming this comment becomes unnecessary and can be removed.